bundles/binding/org.openhab.binding.sagercaster/src/main/java/org/openhab/binding/sagercaster/internal/SagerCasterBinding.java

```diff
@@ -0,0 +1,245 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.sagercaster.internal;
+
+import static org.apache.commons.lang.StringUtils.isBlank;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import org.joda.time.DateTime;
+import org.joda.time.base.AbstractInstant;
+import org.openhab.binding.sagercaster.SagerCasterBindingProvider;
+import org.openhab.core.binding.AbstractBinding;
+import org.openhab.core.items.GenericItem;
+import org.openhab.core.items.Item;
+import org.openhab.core.items.ItemRegistry;
+import org.openhab.core.items.ItemRegistryChangeListener;
+import org.openhab.core.items.StateChangeListener;
+import org.openhab.core.library.types.DecimalType;
+import org.openhab.core.library.types.OnOffType;
+import org.openhab.core.library.types.StringType;
+import org.openhab.core.persistence.HistoricItem;
+import org.openhab.core.persistence.extensions.PersistenceExtensions;
+import org.openhab.core.types.State;
+import org.osgi.framework.BundleContext;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+	
+
+/**
+ * Sager Weather Caster Binding implementation
+ * 
+ * @author Gaël L'hopital
+ * @since 1.7.0
+ */
+public class SagerCasterBinding extends AbstractBinding<SagerCasterBindingProvider> implements ItemRegistryChangeListener, StateChangeListener {
+
+	private static final Logger logger = LoggerFactory.getLogger(SagerCasterBinding.class);
+	private static SagerWeatherCaster sagerWeatherCaster = null;
+	
+	private ItemRegistry itemRegistry;
+	
+	static final String WINDBEARING = "windbearing";
+	static final String SEALEVELPRESSURE = "sealevelpressure";
+	static final String CLOUDLEVEL = "cloudlevel";
+	static final String RAINING = "raining";
+	static final String WINDSPEED = "windspeed";
+	static final String LATITUDE = "latitude";
+	
+	Map<String, String> itemNames = new HashMap<String, String>();	// Link between configuration name and item name
```

Comment:
please rename this map to something more meaningful. If it links itemNames to some keys then rename it to "itemNameToConfigKeyMap" or alike